keyboards/wsk/tkl30/config.h

```diff
@@ -0,0 +1,59 @@
+#ifndef CONFIG_H
```

Comment:
QMK uses the `#pragma once` include guard, instead of `#ifndef`/`#define`/`#endif`.

```suggestion
#pragma once
```